CHANGELOG.md

```diff
@@ -1387,1932 +1387,4 @@ _Changes since Cortex `1.10.0`._
-* [ENHANCEMENT] Upgraded Prometheus to 2.12.0 and Alertmanager to 0.19.0. #1597
-* [ENHANCEMENT] Cortex is now built with Go 1.13 #1675, #1676, #1679
-* [ENHANCEMENT] Many optimisations, mostly impacting ingester and querier: #1574, #1624, #1638, #1644, #1649, #1654, #1702
-
-Full list of changes: <https://github.com/cortexproject/cortex/compare/v0.2.0...v0.3.0>
-
-## Cortex 0.2.0 / 2019-09-05
-
-This release has several exciting features, the most notable of them being setting `-ingester.spread-flushes` to potentially reduce your storage space by upto 50%.
-
-* [CHANGE] Flags changed due to changes upstream in Prometheus Alertmanager #929:
-  * `alertmanager.mesh.listen-address` is now `cluster.listen-address`
-  * `alertmanager.mesh.peer.host` and `alertmanager.mesh.peer.service` can be replaced by `cluster.peer`
-  * `alertmanager.mesh.hardware-address`, `alertmanager.mesh.nickname`, `alertmanager.mesh.password`, and `alertmanager.mesh.peer.refresh-interval` all disappear.
-* [CHANGE] --claim-on-rollout flag deprecated; feature is now always on #1566
-* [CHANGE] Retention period must now be a multiple of periodic table duration #1564
-* [CHANGE] The value for the name label for the chunks memcache in all `cortex_cache_` metrics is now `chunksmemcache` (before it was `memcache`) #1569
-* [FEATURE] Makes the ingester flush each timeseries at a specific point in the max-chunk-age cycle with `-ingester.spread-flushes`. This means multiple replicas of a chunk are very likely to contain the same contents which cuts chunk storage space by up to 66%. #1578
-* [FEATURE] Make minimum number of chunk samples configurable per user #1620
-* [FEATURE] Honor HTTPS for custom S3 URLs #1603
-* [FEATURE] You can now point the query-frontend at a normal Prometheus for parallelisation and caching #1441
-* [FEATURE] You can now specify `http_config` on alert receivers #929
-* [FEATURE] Add option to use jump hashing to load balance requests to memcached #1554
-* [FEATURE] Add status page for HA tracker to distributors #1546
-* [FEATURE] The distributor ring page is now easier to read with alternate rows grayed out #1621
-
-## Cortex 0.1.0 / 2019-08-07
-
-* [CHANGE] HA Tracker flags were renamed to provide more clarity #1465
-  * `distributor.accept-ha-labels` is now `distributor.ha-tracker.enable`
-  * `distributor.accept-ha-samples` is now `distributor.ha-tracker.enable-for-all-users`
-  * `ha-tracker.replica` is now `distributor.ha-tracker.replica`
-  * `ha-tracker.cluster` is now `distributor.ha-tracker.cluster`
-* [FEATURE] You can specify "heap ballast" to reduce Go GC Churn #1489
-* [BUGFIX] HA Tracker no longer always makes a request to Consul/Etcd when a request is not from the active replica #1516
-* [BUGFIX] Queries are now correctly cancelled by the query-frontend #1508
+## [Cortex 1.10.0 CHANGELOG](https://github.com/cortexproject/cortex/blob/master/CHANGELOG.md#1100--2021-08-03)
```

Comment:
Nit, who knows how the future will look like, and whether:
- Cortex will move the file
- Cortex will rename `master` 
- Cortex will move to Gitlab for some reason.

I'd point here to a permalink of current main in Mimir:

```suggestion
## [Cortex 1.10.0 CHANGELOG](https://github.com/grafana/mimir/blob/a13959db5d38ff65c2b7ef52c56331d2f4dbc00c/CHANGELOG.md#cortex-1100--2021-08-03)
```

Reply:
Good suggestion.

Reply:
When the link changes, we can always fix it :) Personally I think pointing to Cortex changelog would make more sense.